### DISCUSSION TITLE
How to migrate existing data from non-operator Kafka cluster in k8s to a new Strimzi operated Kafka cluster

### DISCUSSION THREAD

**Discussion by milon619** [strimzi/strimzi-kafka-operator#4233] [Q&A] [answered]
I have a Kafka cluster running in k8s and it's deployed as a statefulset. Its not managed by any operator. I want to migrate its data (topics, partitions, messages, etc) to a new Kafka cluster that I am going to deploy using the Strimzi Kafka operator. Any idea on how to perform this is highly appreciated.

**Comment by scholzj** [comment, marked as answer]
I guess you have several options. For example:
* Deploy a new cluster and use Mirror Maker to mirror the data
* Try to either re-use the existing volumes / or use the data from them for them and copy them to the right path for Strimzi

It is always a bit individual ... depends on your environment, what do you use today, how big outage can you afford etc.

**Reply by milon619** [reply]
Okay, I am going to try mirror-maker. Thanks for your input 👍
